network-api/networkapi/buyersguide/factory.py

```diff
@@ -111,18 +112,49 @@ def set_image(self, create, extracted, **kwargs):
 
     worst_case = Faker('sentence')
 
+    signup_requires_email = LazyFunction(get_extended_yes_no_value)
```

Comment:
do we wanna add `user_friendly_privacy_policy` to `ProductFactory` as well?

Reply:
might as well.